tests/test_crud_immunisation_api.py

```diff
@@ -342,7 +348,10 @@ def assert_is_filtered(imms):
         assert all(performer["actor"]["identifier"]["system"] == "https://fhir.nhs.uk/Id/ods-organization-code"
                    for performer in imms["performer"])
 
-    assert_is_not_filtered(created_imms)
+    if is_restricted:
```

Comment:
Created will never be filtered because it's the original full object which we need to compare against the retrieved one. I think the work to not return the object on create has scuppered this test. Let's talk in dev stand up tomorrow.